### DISCUSSION TITLE
Batched faster-whisper sometimes transcribes audio files into transcripts with incorrect timestamps of words

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#1005] [General]
When I ran Batched faster-whisper on a lot of audio files, I found that Batched faster-whisper sometimes transcribes audio files into transcripts with incorrect timestamps of words.
Can you please give me some advice on this issue? Thank you.

**Comment by toanhuynhnguyen** [comment]
I hope Batched faster-whisper can process with sentence/word timestamp accuracy like WhisperX. Does anyone have any suggestion or solution?
